packages/popover/src/vaadin-popover.d.ts

```diff
@@ -96,14 +90,14 @@ declare class Popover extends PopoverPositionMixin(
   static setDefaultHoverDelay(hoverDelay: number): void;
 
   /**
-   * String used to label the overlay to screen reader users.
+   * String used to label the popover to screen reader users.
```

Comment:
We could probably deprecate `accessibleName` and `accessibleNameRef` or even remove them in favor of regular `aria-label` and `aria-labelledby`, respectively (and then just remove overrides from the Flow counterpart).